### PR TITLE
Stop removing the extension from module ID

### PR DESCRIPTION
**Pull request by bobisjan**
This PR stops removing the `.hbs` extension from `moduleId`, which aligns implementation with the README now.

Test for EditorConfig shows that `[*.hbs]` section is not taken into account and `insert_final_newline` is `true` within resolved configuration.

### CODE REVIEW

**Comment by rwjblue** [comment]
Hmm. I do think we need to stop removing the extension (future work will include supporting linting within .js / .ts files, for example).

**Comment by bobisjan** [comment]
I've updated PR to stop removing extension, should update PR's title?

**Comment by bobisjan** [comment]
@rwjblue, PR now targets master since next was merged recently  🎉, would you mind taking a look?

**Comment by bobisjan** [comment]
Rebased, should be ready now ☺️.

**Comment by rwjblue** [comment]
I think we can’t quite change this. I’d like to add `filePath` and keep setting `moduleId` as we are now. All of the internal references to `moduleId` would be updated to first check `filePath`, then we will deprecate using `moduleId` (when we deprecate ember-cli-template-lint).

What do you think @bobisjan?

**Comment by rwjblue** [comment]
I’d like to see this land as a precursor before https://github.com/ember-template-lint/ember-template-lint/pull/1117 (which currently introduces `filePath` along with a few other things).

**Comment by bobisjan** [comment]
@rwjblue, sounds good 👍, I’ll switch to `filePath` here.

File: test/acceptance/cli-test.js [hunk 1]
```diff
@@ -430,6 +430,18 @@ describe('ember-template-lint executable', function() {
...
+    it('should exit without error and any console output', function() {
+      let result = run(['app/templates/application.hbs'], {
+        cwd: './test/fixtures/with-editorconfig',
+      });
+
+      expect(result.exitCode).toEqual(0, 'exits without error');
+      expect(result.stdout).toBeFalsy();
+      expect(result.stderr).toBeFalsy();
+    });
+  });
```
Comment (rwjblue) [inline test/acceptance/cli-test.js]: Would you mind leveraging the (newish) fixturify setup instead of adding new fixture files?

Comment (bobisjan) [reply test/acceptance/cli-test.js]: Yes, should I extend `without-errors` fixture with `.editorconfig`?

**Comment by suchitadoshi1987** [comment]
I believe there are cases in `test/acceptance/public-api-test.js` test as well which might need to be updated as well for `moduleId` For eg: (https://github.com/ember-template-lint/ember-template-lint/blob/master/test/acceptance/public-api-test.js#L368)

File: test/acceptance/cli-test.js (hunk 1 above)
Comment (rwjblue) [reply test/acceptance/cli-test.js]: Sorry for the confusion, I’m saying that we should just not use fixtures at all. I’ll drum up an example for you from the recent work in 2.0.

Comment (rwjblue) [reply test/acceptance/cli-test.js]: https://github.com/ember-template-lint/ember-template-lint/blob/4279eb30f9c7c20954951d26b74e1b34dcdb5b89/test/unit/get-config-test.js#L22

Comment (bobisjan) [reply test/acceptance/cli-test.js]: Thanks, I got it and will update later.

File: bin/ember-template-lint.js [hunk 2]
```diff
@@ -119,10 +117,11 @@ function run() {
   }
 
   for (let relativeFilePath of filesToLint) {
-    let filePath = path.resolve(relativeFilePath);
-    let fileName = relativeFilePath === STDIN ? filePathFromArgs : relativeFilePath;
-    let moduleId = fileName.slice(0, -4);
-    let fileErrors = lintFile(linter, filePath, moduleId, fix);
+    let resolvedFilePath = path.resolve(relativeFilePath);
+    let toRead = resolvedFilePath === STDIN ? process.stdin.fd : resolvedFilePath;
```
Comment (rwjblue) [inline bin/ember-template-lint.js]: 👍 this is a really nice bit of cleanup, thank you!

**Pull request merged**